examples/IF_and_HH_responses_to_injection.py

```diff
@@ -0,0 +1,119 @@
+# encoding: utf-8
```

Comment:
This file should not be part of this pull request. You will need to remove it from your branch.